addons/web/__manifest__.py

```diff
@@ -469,7 +469,23 @@
         "web.chartjs_lib" : [
             '/web/static/lib/Chart/Chart.js',
             '/web/static/lib/chartjs-adapter-luxon/chartjs-adapter-luxon.js',
-        ]
+        ],
+        "web._fullcalendar_lib" : [
```

Comment:
why is it here?

Reply:
this is used in enterprise ( in the other PR ) https://github.com/odoo/enterprise/pull/49466